docs/source/usage/picmi/intro.rst

```diff
@@ -100,10 +136,14 @@ Parameters/Methods prefixed with ``picongpu_`` are PIConGPU-exclusive.
   - ``picongpu_template_dir``:
     Specify the template dir to use for code generation,
     please refer to :ref:`the documentation on the matter for details <picmi-custom-generation>`
+  - ``picongpu_custom_user_input``:
+    pass custom user input to the code generation. this may be used in conjuntion with custom templates to change the code
+    generation.
+    please refer to `missing Reference` for the docuemntation on using custom input
```

Comment:
Missing a reference

Reply:
It would be great if the warm plasma example (or some other piece of code) shows how the picongpu extensions are used, but it is not necessary for this PR

Reply:
One of the next PR will replace the example with the LWFA example, using most picongpu extensions